src/main/resources/config/liquibase/changelog/20201111141743_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
You need to include this file in `liquibase/master.xml`

Reply:
Done